src/mpyl/schema/mpyl_config.schema.yml

```diff
@@ -481,6 +483,26 @@ definitions:
     dependencies:
       accessKeyId: [ secretAccessKey ]
       secretAccessKey: [ accessKeyId ]
+  Camunda:
+    type: object
+    properties:
```

Comment:
we don't want the `all` option here?

Reply:
sorry, I am a little bit confused. I don't fully understand the use of all. if I add an `all` option (pr, test, acceptance, production, `all`). what this option should do? deploy to all Camunda environments or just a specific env? I don't think we want either